### ISSUE TITLE
org.ksoap2.serialization.SoapObject.getPropertyInfo() is not returning property value

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josaluntu/ksoap2-android#10]
```
What steps will reproduce the problem?
Call org.ksoap2.serialization.SoapObject.getPropertyInfo() with an
PropertyInfo object.

What is the expected output? What do you see instead?
The PropertyInfo.value field is null whereas other fields are set correctly.

What version of the product are you using? On what operating system?
2.3

```

Original issue reported on code.google.com by `siva.cha...@gmail.com` on 18 Nov 2009 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
Please provide a test case that fails against the latest version on github.
```

Original comment by `mosa...@gmail.com` on 12 Oct 2010 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
Also this might not be a problem anymore with the latest version (2.5.2) since 
a whole bunch of improvements made it in already.
```

Original comment by `mosa...@gmail.com` on 2 Nov 2010 at 3:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `mosa...@gmail.com` on 4 Nov 2010 at 7:59
- Changed state: **Incomplete**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mosa...@gmail.com` on 5 Jul 2011 at 6:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Will be released with 2.5.7
```

Original comment by `mosa...@gmail.com` on 5 Jul 2011 at 6:24
